tools/cli/domainCommands.go

```diff
@@ -282,6 +283,72 @@ func (d *domainCLIImpl) UpdateDomain(c *cli.Context) {
 	}
 }
 
+// FailoverDomains is used for managed failover all domains with domain data IsManagedByCadence=true
+func (d *domainCLIImpl) FailoverDomains(c *cli.Context) {
+	targetCluster := getRequiredOption(c, FlagActiveClusterName)
+
+	domains := d.getAllDomains()
+	shouldFailover := func(domain *shared.DescribeDomainResponse) bool {
+		isDomainNotActiveInTargetCluster := domain.ReplicationConfiguration.GetActiveClusterName() != targetCluster
+		return isDomainNotActiveInTargetCluster && isDomainFailoverManagedByCadence(domain)
+	}
+	var succeedDomains []string
+	var failedDomains []string
+	for _, domain := range domains {
```

Comment:
This seems to be a very dangerous operation. How about ask for confirmation from operator before proceeding. For example, print the list of domains that are about to failover so that the operator can at least do some verification?

Reply:
updated. add a simple prompt